packages/ui/src/LazyItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FunctionComponent, HTMLAttributes, useEffect, useRef, useState } from 'react'
+import styled from 'styled-components'
+
+import useIntersectionObserver from 'ui/src/hooks/useIntersectionObserver'
+
+type Props = {
+  defaultHeight: string
+} & HTMLAttributes<HTMLTableRowElement>
```

Comment:
cool! If useful we could make this more generic (it doesn't need to be limited to TableRowElement)